sql/src/main/java/io/crate/expression/symbol/SymbolVisitors.java

```diff
@@ -22,12 +22,22 @@
 
 package io.crate.expression.symbol;
 
+import java.util.List;
 import java.util.function.Predicate;
 
 public class SymbolVisitors {
 
     private static final AnyPredicateVisitor ANY_VISITOR = new AnyPredicateVisitor();
 
+    public static boolean any(Predicate<? super Symbol> symbolPredicate, List<Symbol> symbols) {
```

Comment:
Could change the second argument to `Iterable<? extends Symbol>`. The function doesn't require more and it would make it usable for more cases.